roxie/ccd/ccddali.cpp

```diff
@@ -600,7 +600,7 @@ class CRoxieDaliHelper : implements IRoxieDaliHelper, public CInterface
             {
                 if (traceLevel > 1)
                     DBGLOG("checkClonedFromRemote: Resolving %s in legacy mode", _lfn);
-                Owned<IDistributedFile> cloneFile = resolveLFN(foreignLfn, cacheIt, AccessMode::tbdRead, isPrivilegedUser);
+                Owned<IDistributedFile> cloneFile = resolveLFN(foreignLfn, cacheIt, AccessMode::readMeta, isPrivilegedUser);
```

Comment:
Not sure I quite understand when you would use readMeta - if you are then going to read the data there's a danger that the meta you get refers to the wrong plane?